### ISSUE TITLE
[4.0.0-beta.7] React-hot-loader: wrong configuration

### ISSUE THREAD

**Issue by katmai7** [gaearon/react-hot-loader#750]
After updating from 4.0.0-beta6 to 4.0.0-beta7 version, I've gotten an error `React-hot-loader: wrong configuration`.

It looks like I import and use hot loader correctly
```
import React from 'react';
import { hot } from 'react-hot-loader';
// other imports

function renderRouter() {
  return (
    <Router5Provider router={router}>
      <ReactRouter history={history}>
        {routesFactory(store)}
      </ReactRouter>
    </Router5Provider>
  );
}

export default hot(module)(renderRouter);
```
I've tried to import hot-loader before React, but nothing changed.

React Hot Loader version: 4.0.0-beta.7
1. `node -v`: 8.9.1
2. `npm -v`: 5.6.0

3. Operating system: OS X El Capitan 10.11.6
4. Browser and version:  Chrome 63.0.3239.84

**Comment by neoziro** [comment]
@katmai7 can you try with [v4.0.0-beta.8](https://github.com/gaearon/react-hot-loader/releases/tag/v4.0.0-beta.8)?

**Comment by theKashey** [comment]
@katmai7 - it should contain more extended information, written by console.error just before triggering an error.
The problem may be not in this file, but somewhere before, or after it - RHL just detects component which uses the unpatched version of React, and report about it.

How to solve completely - `import 'react-hot-loader'` in the index.js, ie before any other your file. Just import, nothing more.
Anyway - it is better to trace the problem or provide some reproducible demo, and we will trace it.

**Comment by neoziro** [comment]
@theKashey you mean `import 'react-hot-loader/patch'` I think.

**Comment by theKashey** [comment]
react-hot-loader will import index.dev, it will import AppContainer.dev, it will import patch.dev :)
So you can just import `react-hot-loader`.

**Comment by neoziro** [comment]
Should be definitively fixed in [4.0.0-beta.9](https://github.com/gaearon/react-hot-loader/releases/tag/v4.0.0-beta.9).
